apps/andi/lib/andi_web/live/helpers/sorting_helpers.ex

```diff
@@ -0,0 +1,29 @@
+defmodule AndiWeb.Helpers.SortingHelpers do
+  @moduledoc """
+  Helpers for sorting a list of maps by a field. Includes the ability to mix structs into the sorting (at least DateTime)
+  """
+  def sort_list_by_field(list, field, dir \\ "asc") do
```

Comment:
I like this, we might be able to move the StructTools sort by sequence function here too. We made it public in our PR but it's private in master so yaidontknow

Reply:
either way - mostly I was wanting to do some more extensive testing and it was painful to have to mock up entire datasets to do it in the table_test